server/src/internalClusterTest/java/org/elasticsearch/action/termvectors/GetTermVectorsIT.java

```diff
@@ -1048,9 +1047,9 @@ public void testTermVectorsWithIgnoredField() throws IOException, InterruptedExc
         // add a doc with a bad long field
         indexRandom(true, prepareIndex("index").setId("1").setSource("{\"field\":\"foo\"}", XContentType.JSON));
 
-        // do a tv request for all fields, _ignored should be returned
+        // do a tv request for all fields, _ignored should not be returned as a result of not being stored but just having doc values
         TermVectorsResponse resp = client().prepareTermVectors("index", "1").setSelectedFields("*").get();
-        assertThat(resp.getFields().terms("_ignored").size(), greaterThan(0L));
```

Comment:
I will need to restore this test to the original and have the field type call `setStoreTermVectors(true)` to make the _ignored field store also term vectors.